src/domains/ecommerce-common/00_alerts.tf

```diff
@@ -298,4 +298,48 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
-}
\ No newline at end of file
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_notifications_service_availability" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-notifications-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, azurerm_monitor_action_group.ecommerce_opsgenie[0].id]
+    email_subject          = "[eCommerce] Availability Alert"
```

Comment:
please update `email_subject` with a more descriptive phrase regarding the availability of the notification-service

Reply:
Done here e7023e6aa5e5f1cfa54dd81c21aa53461daed3a3